example.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 
 let rosjs = require('./index.js');
+const std_msgs = rosjs.requires('std_msgs').msg;
+const ListCameras = rosjs.requires('baxter_core_msgs').srv.ListCameras;
```

Comment:
Do you really want your test to depend on `baxter_core_msgs`? Perhaps something similar to the `add_two_ints` service example?